dali_tf_plugin/dali_dataset.h

```diff
@@ -75,6 +76,24 @@ class DALIDatasetOp : public tensorflow::data::DatasetOpKernel {
     bool enable_memory_stats;
   };
 
+  // TODO(klecki): I know the name Inputs is not ideal, but we actually get them
+  // in compute from Op inputs.
+  struct Inputs {
+    std::vector<tensorflow::data::DatasetBase *> inputs;
+  };
+
+  // Those are the static Attrs describing inputs
+  struct InputDef {
```

Comment:
....if these are Attrs, then maybe
```suggestion
  struct InputAttrs {
```
?

Reply:
Why not, done.